common/src/test/java/gov/cms/ab2d/common/service/CacheServiceImplTest.java

```diff
@@ -58,17 +56,23 @@
     @BeforeEach
     void setUp() {
         coverageRepo.deleteAll();
+        coverageSearchEventRepo.deleteAll();
+        coverageSearchRepo.deleteAll();
```

Comment:
deleteAll() should be in the teardown().

Reply:
I'll fix that in the next PR coming up